package.json

```diff
@@ -1269,6 +1269,65 @@
                 "command": "vscode-neovim.complete-selection-cmdline",
                 "key": "tab",
                 "when": "neovim.mode == cmdline_normal || neovim.mode == cmdline_insert || neovim.mode == cmdline_replace"
+            },
+            {
+                "key": "ctrl+w",
+                "command": "-workbench.action.switchWindow"
```

Comment:
What is this for? I don't think this is a vscode default

Reply:
<img width="855" alt="Screenshot 2023-07-03 at 12 28 46" src="https://github.com/vscode-neovim/vscode-neovim/assets/24259245/74960aac-32c6-41bb-bd09-c57b767bee53">

It definitely is and it intercepts.

Reply:
Strange, it isn't for me. Nothing is bound to switchWindow. Maybe a macos thing?

![image](https://github.com/vscode-neovim/vscode-neovim/assets/16546293/6e5678b8-3ca5-4ad6-9f97-9e0d7eb57f0b)

Reply:
@theol0403 Looks like it. [This gentleman](https://stackoverflow.com/a/52289040/1540350) also uses MacOS I think.